CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/LabelFontsOptionsPanelController.java

```diff
@@ -136,7 +136,7 @@ public void cancel() {
 
     @Override
     public boolean isValid() {
-        final LabelFontsOptionsPanel labelFontsOptionsPanel = getPanel();
+        getPanel();
```

Comment:
I think that this Method is actually incomplete but at the moment it retrieves the panel to test if it exists.